.github/CODEOWNERS

```
@@ -1 +1,4 @@
-* @siriak @acylam @eshom @Panquesito7
+* @siriak @acylam @eshom
+*.md              @Panquesito7
+*.github/         @Panquesito7
+*LICENSE          @Panquesito7
```

```suggestion
.github/          @Panquesito7
LICENSE           @Panquesito7
```
Shouldn't it be like that?